README.md

```diff
@@ -70,6 +70,22 @@ transformations_over_time = train(transformations, X, y, splitter)
 pred = backtest(transformations_over_time, X, y, splitter)
 ```
 
+With the `train_evaluate` and `evaluate` helper function (requires [krisi](https://github.com/dream-faster/krisi)):
+
+```python
+from fold import train_evaluate
+X
+y = X.squeeze()
+
+splitter = ExpandingWindowSplitter(initial_train_window=400, step=400)
+transformations = [
+  DummyRegressor(strategy="constant", constant=0),
+  OnlyPredictions(),
+]
+scorecard, prediction, trained_transformations =  train_evaluate(transformations, X, y, splitter)
+
```

Comment:
no I think that's fine! could you remove the previous, oudated example?

Reply:
I have updated it, so now there is both? I think _just_ having this is not great as it is dependent on `krisi`, but then this is also good to have here because its a shorter version.

Reply:
Actually I'll make krisi optional in the abstractions aswell.